swebench/harness/log_parsers/python.py

```diff
@@ -249,6 +251,100 @@ def parse_log_matplotlib(log: str, test_spec: TestSpec) -> dict[str, str]:
             test_status_map[test_case[1]] = test_case[0]
     return test_status_map
 
+###################
+# copy paste our parsers for our instances.    
+
+def parser_pytest(
+    log: str,
+) -> dict[str, str]:
+    
+    test_result_dict: dict[str, str] = {}
+
+    # check for pytest
+    if log.find("short test summary info") != -1:
```

Comment:
Please always use:
``` "short test summary info" in log```
Instead of this find construction. Its faster and more readbale.